src/proj.h

```diff
@@ -182,27 +262,93 @@ PJ_OBSERVATION pj_apply (PJ *P, enum pj_direction direction, PJ_OBSERVATION obs)
 /* Measure internal consistency - in forward or inverse direction */
 double pj_roundtrip(PJ *P, enum pj_direction direction, int n, PJ_OBSERVATION obs);
 
-
 int pj_show_triplet (FILE *stream, const char *banner, PJ_TRIPLET point);
 
-/* Constructor for the OBSERVATION object */
+PJ *pj_pj (PJ_CONTEXT *ctx, char *definition);
+PJ *pj_pj_argv (PJ_CONTEXT *ctx, int argc, char **argv);
+
+
+/* Initializer for the OBSERVATION object */
 PJ_OBSERVATION pj_observation (
     double x, double y, double z, double t,
     double o, double p, double k,
     int id, unsigned int flags
 );
 
+#ifndef PJ_OBSERVATION_C
+extern const PJ_OBSERVATION pj_observation_error;
+extern const PJ_OBSERVATION pj_observation_null;
+#endif
+
 #ifndef TODEG
 #define TODEG(rad)  ((rad)*180.0/M_PI)
 #endif
 #ifndef TORAD
 #define TORAD(deg)  ((deg)*M_PI/180.0)
 #endif
 
-#endif /* ndef PROJECTS_H */
+
+
+
+/* This is mostly a direct copy of (parts of) the proj_api header, but with cleaned up name space */
+
+#define PJ_VERSION 493
```

Comment:
I have changed it now, so it lives in proj_api.h only - but making it work requires some ugly defines.